composer.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
What is this file ???

Reply:
Auto generate, but i don't know by what. I delete him for the next commit.